src/features/user/Account/History.tsx

```diff
@@ -151,6 +151,16 @@ export default function History({
                       >
                         {t(item[0])}
                       </div>
+                    ) : (
+                      <div
+                        key={item[0]}
+                        className={`${styles.filterButton} ${
+                          filter === item[0] ? styles.selected : ''
+                        }`}
+                        onClick={() => handleSetFilter(item[0])}
+                      >
+                        {t('me:pending')}
```

Comment:
You seem to have duplicated the whole div-tag looking like the same code, is it? Why don't you just make a decision on what translation resource to use, e.g. `{t(item[0] !== 'in-progress' ? item[0] : 'me:pending')}`?

Reply:
Btw. this looks like quite a hack for the backend returning a wrong status, is it? We should add a comment about this for some future developer looking at it!

Reply:
it is going to be fix in the backend , as in frontend we are mapping the response to display  button